values/graphprotocol-indexer-service.yaml

```diff
@@ -0,0 +1,29 @@
+role: indexer-service
+
+replicas: 2
+
+postgres:
+  db: graph-node 
+  host: postgres-postgresql
+  user: postgres
+  # password: "ChangeMe"
+
+config:
+  clientSignerAddress: 0xe1EC4339019eC9628438F8755f847e3023e4ff9c
+  ethereum: 
+    network: rinkeby
+    # url: "ChangeMe"
+    # address: "ChangeMe"
+    # mnemonic: "ChangeMe"
+  graphEndpoints:
+    admin: http://graphprotocol-node-index:8020/
+    status: http://graphprotocol-node-index:8030/graphql
+    query: http://graphprotocol-node-query:8000/
+
+  networkSubgraphEndpoint: https://gateway.testnet.thegraph.com/network
+  subgraphDeployment: QmUBRDENCjFt57vVjFc3D4EQoYqXJffKiyFyuGYB2BN8Wi
+
+# ingress:
```

Comment:
We could omit this for now I think

Reply:
We need ing, cause to participate in testnet we need public endpoint, I'm already created ingress in test cluster using own domain.